### DISCUSSION TITLE
after `winget list` Powershell only show a blue line

### DISCUSSION THREAD

**Discussion by Mannshoch** [microsoft/winget-cli#1903] [Q&A] [answered]
I today freshly installed Winget (App-Installer) from Microsoft store (Version: 1.17.10271.0)
If I run `winget list` it spins shortly and then the command line turns blue, without any output.

`winget source update` instead works without problems

**Comment by ghost** [comment]
related see : [Winget List command takes 10 seconds to complete. #855]( https://github.com/microsoft/winget-cli/issues/855)

Why still bouncing around not fixed ? : Simply Reluctance.

**Reply by Mannshoch** [reply]
I waited now for around two hours. I don't think it's the same Problem.
![grafik](https://user-images.githubusercontent.com/7795891/152746606-9ba545d8-1c04-4664-9c66-f767a8620578.png)

**Reply by Mannshoch** [reply]
I detected the Problem is this source:
winget  https://winget.azureedge.net/cache
If I  rmove this source Winget works.

I suppose the Problem is "Cisco Umbrella"

**Reply by leomelki** [reply]
Hello, do you have any update on how to fix this please ? I have exactly the same problem but getting rid of the winget source is quite problematic

**Reply by Mannshoch** [reply]
I assume that Winget do a SSL Certificate Pining. After I requestet an exception for Winget in my Company it works.
Sometimes some shady Antivirus software do same. In such cases you have to deactivate it.

**Reply by leomelki** [reply]
I made it work by updating [this thing from the Microsoft Store](https://apps.microsoft.com/detail/9NBLGGH4NNS1?hl=en-us&gl=US) if anyone get this error.

**Reply by mdanish-kh** [reply]
@Mannshoch What version of WinGet are you using? You can use `winget -v` to find out. This is likely related to CDN migration that broke some older versions of WinGet.
- https://github.com/microsoft/winget-cli/issues/3652

Please try to upgrade to the latest client either from Microsoft Store, [GitHub Releases](https://github.com/microsoft/winget-cli/releases) or this URL https://aka.ms/getwinget.

**Reply by Mannshoch** [reply, marked as answer]
I could not say If this still is the same state like in April 2022. As I said since I reported the problem to the IT department, my problem was solved.

**Reply by dustojnikhummer** [reply]
This is still an issue on clean Windows 11 installs. Is there a way to upgrade it through a powershell script, rather than waiting 20 minutes for Windows Store to decide to update it?

**Reply by mdanish-kh** [reply]
@dustojnikhummer Have you tried using

```
Add-AppxPackage -Path "https://aka.ms/getwinget"
```

If some dependencies are missing, you can try using the ["Sandbox script"](https://learn.microsoft.com/en-us/windows/package-manager/winget/#install-winget-on-windows-sandbox)

**Reply by jonashilmersson** [reply]
Using the **Add-AppxPackage** method above was the only thing working for me to get winget to work in a clean Windows 11 23H2. I wasn't allowed to update winget using the installation bundle or by updating the App Installer via MS Store.

**Reply by ericnordebaeck** [reply]
@mdanish-kh 's reply should be the selected answer. Confirmed that this fixed the issue for us as well.
